src/test/java/uk/nhs/hee/tis/trainee/sync/service/DataRequestServiceTest.java

```diff
@@ -0,0 +1,37 @@
+package uk.nhs.hee.tis.trainee.sync.service;
+
+import com.amazonaws.services.sqs.AmazonSQS;
+import com.amazonaws.services.sqs.model.SendMessageRequest;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.junit.jupiter.MockitoExtension;
+import static org.mockito.Mockito.*;
+
+@ExtendWith(MockitoExtension.class)
```

Comment:
Don't need to specify this if we're creating mocks manually and not using annotations.